controllers/reservation.controllers.js

```diff
@@ -519,31 +517,39 @@ const submitReserve = async ({
     let day = start;
 
     while (day <= end) {
-      days.push(day.toDate());
+      const dayString = day.format('dddd');
+      if (dayString !== 'Saturday' && dayString !== 'Sunday') {
```

Comment:
both this conditional and the "Saturday" "Sunday" values are getting reused in lines above. I think we could either make this conditional a new variable that these ifs can reuse, or move the day strings to a constants file. Or both :)